.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# nf-core/modules: Contributing Guidelines
+
+Hi there!
+Many thanks for taking an interest in improving nf-core/modules.
+
+We try to manage the required tasks for nf-core/modules using GitHub issues, you probably came to this page when creating one.
+Please use the pre-filled template to save time.
+
+However, don't be put off by this template - other more general issues and suggestions are welcome!
+Contributions to the code are even more welcome ;)
+
+> If you need help using or modifying nf-core/modules then the best place to ask is on the nf-core Slack [#modules](https://nfcore.slack.com/channels/modules) channel ([join our Slack here](https://nf-co.re/join/slack)).
+
+## Contribution workflow
+
+If you'd like to write some code for nf-core/modules, the standard workflow is as follows:
+
+1. Check that there isn't already an issue about your idea in the [nf-core/modules issues](https://github.com/nf-core/modules/issues) to avoid duplicating work
+    * If there isn't one already, please create one so that others know you're working on this
+2. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the [nf-core/modules repository](https://github.com/nf-core/modules) to your GitHub account
+3. Make the necessary changes / additions within your forked repository following [Pipeline conventions](#pipeline-contribution-conventions)
```

Comment:
I don't know whether this point it is needed, in any case the reference I think the reference is not working